dll/cpl/input/input.c

```diff
@@ -39,22 +39,13 @@ static int CALLBACK
 PropSheetProc(HWND hwndDlg, UINT uMsg, LPARAM lParam)
 {
     // NOTE: This callback is needed to set large icon correctly.
-    // FIXME: Handle leaks
-    HICON hIcon, hIconSmall;
+    HICON hIcon;
     switch (uMsg)
     {
         case PSCB_INITIALIZED:
         {
             hIcon = LoadIconW(hApplet, MAKEINTRESOURCEW(IDI_CPLSYSTEM));
             SendMessageW(hwndDlg, WM_SETICON, ICON_BIG, (LPARAM)hIcon);
-
-            hIconSmall = (HICON)LoadImageW(hApplet,
-                                           MAKEINTRESOURCEW(IDI_CPLSYSTEM),
-                                           IMAGE_ICON,
-                                           GetSystemMetrics(SM_CXSMICON),
-                                           GetSystemMetrics(SM_CYSMICON),
-                                           0);
-            SendMessageW(hwndDlg, WM_SETICON, ICON_SMALL, (LPARAM)hIconSmall);
```

Comment:
Why did you revert this?

Reply:
This looks like it was not intentionally reverted, but was a merge-oopsie.
Those code-pieces was merged a day ago just and was committed by Katayama in
https://git.reactos.org/?p=reactos.git;a=commit;h=976d48baeb1a98d620107885022b44b24bad77a1

Reply:
This is not needed. All other CPLs that use `PROPSHEETHEADER` don't do this.
The small icon is already set by `PROPSHEETHEADER` structure.

The problem comes from the applet index.

Reply:
Not sure why that happens. Maybe because the index returned by `CPL_INQUIRE` can be a nagative value?

Reply:
@TAN-Gaming saying is correct.
https://github.com/reactos/reactos/blob/3647f6a5eb633b52ef4bf1db6e43fc2b3fc72969/dll/win32/comctl32/propsheet.c#L3553

```c
SendMessageW(hwnd, WM_SETICON, 0, (LPARAM)hIcon);
```
and
```c
#define ICON_SMALL             0
```

Reply:
If you find a solution that does satisfy you and Katayama, and the icon remains in working state after that, then I am ofc fine with whatever you'll decide together.
>>The problem comes from the applet index.
I don't understand.
Does the icon reliably show with the code as it currently is in your PR?

Reply:
If the applet index is negative, then what will happen?
Why is the applet index negative?

Reply:
BEFORE and AFTER screenshots please.

Reply:
@JoachimHenze 

> Does the icon reliably show with the code as it currently is in your PR?

Yes, other CPLs also use `int`.